Chinese/Interstellar/Interstellar.md

```diff
@@ -6,4 +6,4 @@
 
 当画面来到最后一幕：五指山下，悟空蓬头垢面，三藏袈裟翩翩，正站在他面前，对他说了一句：“荆轲刺秦王”。
 
-悟空从中回过神来，撕心裂肺地喊出：“穿山甲到底说了什么！”。然后突然听到台下有人大叫：“王尼玛我要给你生猴子！”。
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.